src/test/java/LoginTests.java

```diff
@@ -1,25 +1,36 @@
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
+import POM.Pages.HomePage;
+import POM.Pages.LoginPage;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-import java.time.Duration;
+
 
 public class LoginTests extends BaseTest {
+
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void registrationNavigation() throws InterruptedException{
+        LoginPage loginPage = new LoginPage(getDriver());
+        Thread.sleep(3000);
+        loginPage.getRegistered();
+        Thread.sleep(3000);
+        Assert.assertTrue(loginPage.registerButton().isDisplayed());
+    }
 
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+    @Test
+    public void loginValidCredentials()throws InterruptedException{
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        Thread.sleep(3000);
+        loginPage.enterEmail("angel.ayala@testpro.io").enterPassword("school!sc0").clickSubmit();
+        Thread.sleep(2000);
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.